hub/tx_init/rows/acc_coinbase_warm.tex

```diff
@@ -0,0 +1,26 @@
+In accordance with \cite{EIP-3651}, the purpose of the current row is to prewarm the coinbase address:
+\[
+	\left\{ \begin{array}{lclr}
+		\accAddress  \high _{i + \locTxInitWarmCoinbase} & = & \txCoinbase\high _{i + \locTxInitRoffTxn}\\
+		\accAddress  \low  _{i + \locTxInitWarmCoinbase} & = & \txCoinbase\low  _{i + \locTxInitRoffTxn}\\
+		\multicolumn{4}{l}{
+			\accTrimAddress
+			{i}{\locTxInitWarmCoinbase}
+			{\locTxInitRecipientAddressHi}
+			{\locTxInitRecipientAddressLo}} \vspace{2mm} \quad (\trash)\\
+		\multicolumn{3}{l}{\accSameBalance                       {i}{\locTxInitWarmCoinbase}} \\
+		\multicolumn{3}{l}{\accSameNonce                         {i}{\locTxInitWarmCoinbase}} \\
+		\multicolumn{3}{l}{\accSameCode                          {i}{\locTxInitWarmCoinbase}} \\
+		\multicolumn{3}{l}{\accSameDeployment                    {i}{\locTxInitWarmCoinbase}} \\
+		\multicolumn{3}{l}{\accTurnOnWarmth                      {i}{\locTxInitWarmCoinbase}} \\
+		\multicolumn{3}{l}{\accSameMarkedForSelfdestructFlag     {i}{\locTxInitWarmCoinbase}} \\
+		% \multicolumn{3}{l}{\accIsntPrecompile                    {i}{\locTxInitWarmCoinbase}} \\
+		\multicolumn{3}{l}{
+			\standardDomSubStamps {
+				anchorRow = i                               ,
+				relOffset = \locTxInitWarmCoinbase          ,
+				domOffset = 0                               ,
```

Comment:
You have to update the other DOM_OFFSET's, too (getting a +1). To prevent DOM/SUB collisions.